docker-compose.yml

```diff
@@ -86,12 +86,63 @@ services:
           - openmrs-mysql
     volumes:
       - "openmrs-mysql-data:/var/lib/mysql"
+      - "./mysql-dump:/docker-entrypoint-initdb.d"
```

Comment:
I like this idea of providing the mapping to the SQL by default.
However, I would structure the `mysql-dump` folder a bit differently such as:
Something like:
- sqls/mysql/
- sqls/postgres/
- sqls/...

Reply:
Agreed. We can add documentation for the same.